### ISSUE TITLE
'TooManyAttemptsBannedException' from 'pyoverkiz.exceptions'

### ISSUE THREAD

**Issue by MikeLDPT** [iMicknl/ha-tahoma#811]
### Did you read the instructions?

- [X] I have read the [README](https://github.com/iMicknl/ha-tahoma/blob/master/README.md), including the [advanced debugging](https://github.com/iMicknl/ha-tahoma/blob/master/README.md#advanced) section.

### The problem

Hi. I have found that #799 relates to this, and that something has been done to address this error appearing in the logs, but I cannot actually understand what has been done to fix it.
Could you please point me in the right direction for resolving this?  

Thank you

### What version of this integration (ha-tahoma) has the issue?

2.13.0

### What version of Home Assistant Core has the issue?

2022.6.4

### Which gateway / hub do you use?

Connexoon RTS

### Device model

Tahoma Basic

### Home Assistant log

<details><summary>Logs</summary>

```
Logger: homeassistant.config_entries
Source: config_entries.py:749
First occurred: 11:22:06 (2 occurrences)
Last logged: 11:22:06

Error occurred loading configuration flow for integration overkiz: cannot import name 'TooManyAttemptsBannedException' from 'pyoverkiz.exceptions' (/usr/local/lib/python3.9/site-packages/pyoverkiz/exceptions.py)
```

</details>

**Comment by iMicknl** [comment]
Have you tried using Overkiz from core?

**Comment by MikeLDPT** [comment]
No... I have found instruction on "how to" in another issue.... I'll give it a go, thanls

**Comment by iMicknl** [comment]
Great! Could you remove the custom component and just use the Overkiz integration from core, from there we can have a look how to fix it. This repository won't be actively improved anymore.
